feat: initial text

marvinmarnold opened the pull request:
Issue: https://github.com/NebraLtd/maker-starter-app/issues/11
Summary: Need to update assets so that app will have our branding and sensible text.

marvinmarnold commented on app.json:
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "MakerApp",
-  "displayName": "Maker App",
+  "name": "The Nebra Helium App",
```
@vpetersson is there a different official name?

marvinmarnold commented on src/locales/en.ts:
```diff
@@ -51,8 +51,8 @@ export default {
       subtitle:
-        'Lorem ipsum <b><errorText>$HNT</errorText></b> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
-      title: 'Welcome\nto Maker App',
+        'Use this app to onboard your miner for the first time and start earning HNT. <b>WARNING: You must also have the official Helium App installed in order to continue.</b>',
```
![image](https://user-images.githubusercontent.com/1311030/156064824-40b13247-5ae7-4334-86b6-faa928649078.png)

marvinmarnold commented on src/locales/en.ts:
```diff
@@ -167,6 +168,7 @@ export default {
           title: '[Placeholder]',
         },
       ],
+      // TODO
```
![image](https://user-images.githubusercontent.com/1311030/156064966-07abe832-66c9-42e4-91a5-261cecdf8417.png)

marvinmarnold commented on src/makers/example/hotspots.ts:
```diff
@@ -2,12 +2,14 @@ import HotspotIcon from './icon.svg'
 import { MakerHotspot } from '../hotspotMakerTypes'
 import ANTENNAS from './antennas'
 
+// TODO: variants
```
@vpetersson @shawaj we must define all the miner types we support.

- Indoor (Raspberry Pi)
- Outdoor (Raspberry Pi)
- Indoor (ROCK Pi)

What other variants do we want to include in the app? What are the public facing names for these variants? The labels currently say something like "Nebra Helium Hotspot: NEBHNT-HHRK4-915", none of which makes sense to use in this context.

marvinmarnold commented on src/makers/example/hotspots.ts:
```diff
@@ -45,6 +47,7 @@ const ExampleHotspotQR = {
   onboardType: 'QR',
   translations: {
     en: {
+      // Different instructions
```
I need to familiarize myself with this flow, not sure right now

marvinmarnold commented on src/makers/example/index.ts:
```diff
@@ -4,6 +4,6 @@ import hotspots from './hotspots'
   hotspots,
-  id: 99999, // TODO: Get maker id from onboarding server
-  supportEmail: 'support@examplemaker.com',
+  id: 99999, // TODO: Get maker id from onboarding server after creation
```
@vpetersson after you run the node create-maker task, we need to update this with the corresponding ID

marvinmarnold commented:
@posterzh is this the only change I need to revert to fix the name change? https://github.com/NebraLtd/maker-starter-app/pull/21/commits/0d2aba3f08e9c7ec0d8d57f4f0667e626f38b1ad

posterzh commented:
> @posterzh is this the only change I need to revert to fix the name change? [0d2aba3](https://github.com/NebraLtd/maker-starter-app/commit/0d2aba3f08e9c7ec0d8d57f4f0667e626f38b1ad)

Correct.

marvinmarnold commented:
@posterzh @Andrewsopoku just converted this to a real PR. We can still use the convo as a list of todo items, but I want to start getting things merged.

marvinmarnold commented:
@posterzh @Andrewsopoku can you please review?

marvinmarnold commented:
Thanks @Andrewsopoku I hadn't seen that README. I'll create some tickets accordingly.

marvinmarnold commented:
@Andrewsopoku @posterzh  I've updated the PR so it only touches the text. I've updated the original ticket with all the specific UI details: https://github.com/NebraLtd/maker-starter-app/issues/11. Created additional tickets that you can both help with:

- https://github.com/nebraltd/maker-starter-app/issues/25
- https://github.com/nebraltd/maker-starter-app/issues/26
